controller/src/main/java/io/pravega/controller/store/stream/StreamProperty.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
License header format is still not correct (spacing before the URL). Let's just use the header as is, no changes to anything even if the change seems trivial and harmless.

Reply:
fixed